### DISCUSSION TITLE
Feature Request - Option to disable Export to CSV button (Azure Managed Grafana)

### DISCUSSION THREAD

**Discussion by bartvdooren** [grafana/grafana#71624] [Dashboards, Panels & Visualizations]
Hi Reader,

We want to make use of Azure Managed Grafana, however, we are unable to disable the Export to CSV button. This button makes it too easy for users to download the underlying data from the Grafana dashboard, and we want to make the application as tight as possible.

/* Note that we understand that in the response when loading the data, you can still view the data, but we want to not make it that easy for our users */

We notice that this is currently also not a feature from the open source version, resulting in Azure not being able to implement this feature. Therefore the request for this feature to be added. 

In general, removing the share functionality would be even better, then we also automatically have a way to remove the snapshot sharing to snapshots.raintank.io

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82611. Feel free to continue the discussion around this there. Thank you!
